src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -29,6 +30,11 @@
 
 final class ManagingProductsContext implements Context
 {
+    public const SORT_TYPES = ['ascending' => 'asc', 'descending' => 'desc'];
+
+    /** @var ApiClientInterface */
+    private $adminUserClient;
```

Comment:
```suggestion
    private $adminUsersClient;
```
as I wrote yesterday 😢